### DISCUSSION TITLE
به بخش گفتگو خوش آمدید!

### DISCUSSION THREAD

**Discussion by MasterKia** [MasterKia/PersianBlocker#49] [Announcements]
## 👋 خوش آمدید!
  ما از بخش «گفتگو ها»‌ به عنوان مکانی برای ارتباط برقرار کردن با دیگر اعضای جامعه‌مان استفاده میکنیم. ما امیدواریم که:
  * سوالاتی که ذهنتان را مشغول کرده است بپرسید.
  * ایده ها و نظراتتان را به اشتراک بگذارید.
  * با دیگر اعضای جامعه گفتگو و تعامل داشته باشید.
  * از دیگران استقبال کنید و ذهنی پویا داشته باشید. یادتان باشد که این جامعه را ما به کمک هم ساخته‌ایم. 💪.

  هر چه دل تنگت میخواهد بگو.

**Comment by chromer030** [comment]
با سلام و قدردانی از زحمات شما برای Maintain کردن این لیست ارزشمند.

**Reply by ArashRn86** [reply]
خیلی جالبه که  روی سایتها  نوار انداختید که کاربر باید انتخاب کنه  چیو ببینه نه خود سایت
بعد نوار خودتون رو بزور انداختید جلوی چشم ما و نمیشه برش داشت
کاسه داغتر از آش
الان ما مجبوریم شما رو ببینیم به جای چیزی که میخواهیم

**Reply by MasterKia** [reply]
کسی جلوت رو نگرفته، میتونی لیست رو غیرفعال کنی.

**Reply by ArashRn86** [reply]
ایکاش میشد، 
والا وقت نمیذاشتم بیام اینجا برات پیام بذارم
هیچ علامتی برای بستنش نیست

**Comment by SPRZB** [comment]
یه بخش جدا برای گزارش و درخواست بررسی ردیاب‌های سایت‌ها اضافه میکنید؟ (و بالای تمام بحث‌ها قرار بدید؟)

**Reply by MasterKia** [reply]
برای بررسی ردیاب‌ها می‌تونید [Logger رو باز کنید](https://github.com/gorhill/uBlock/wiki/The-logger)، و فیلتر رو روی 3rd-party تنظیم کنید. **_بعدش_** سایت (های) موردنظر رو باز کنید و ببینید چه اتصالاتی اونجا برقرار میشه.
اگه شک داشتید یه Discussion دیگه باز کنید و اون‌جا لینک سایت رو بفرستید تا بررسی بشه.
اون گفتگو رو هم سنجاق خواهیم کرد.

**Reply by SPRZB** [reply]
خب مثلا اون فیلتری که برای ردیاب سایت bama.ir گذاشتید میره توی فیلتر 1st-party.
اینا رو چطور متوجه بشیم تا برای بررسی گزارش بدیم؟

**Reply by MasterKia** [reply]
روشِ کلی‌تر اینه که فیلتر No CSS/Font + Image رو انتخاب می‌کنید (تا Logger خلوت‌تر بشه)، بعد همه درخواست‌ها رو نگاه می‌کنید و اگه درخواستی "به‌نظرتون" مشکوک بود (مثلاً درخواست زدن bama.ir به apm.bama.ir یا درخواست زدن bama.ir به فایلی مثل bama.ir/assets/tracker.js)، میایید اون درخواست رو مسدود می‌کنید؛‌ بعد آزمایش می‌کنید ببینید سایت خراب شده یا نه.
البته اگه علمی‌تر بخوایید عمل کنید باید Inspect Element مرورگر رو باز کنید و به زبونه Network برید و ازونجا ببینید چه اطلاعاتی بین bama.ir و apm.bama.ir رد و بدل شده،‌ بعد تصمیم به مسدود کردنش بگیرید.

**Comment by 123SONIC321** [comment]
> به سوالات مربوط به چگونگی نوشتن فیلتر ها قطعاً پاسخ داده میشود.

سلام و خدا قوت @MasterKia 
خیلی علاقه‌مند هستم که روش‌های مختلف بلاک کردن رو یاد بگیرم (بجای اینکه از المنت پیکر یوبلاک استفاده کنم)
مثلا توی وبسایت توییچ، [صفحه اول](https://twitch.tv/)، یک پلیر داره که با هربار وارد شدن، این پلیر یه استریم رندوم رو شروع به پخش میکنه، تونستم با المنت پیکر از بین ببرمش ولی با ورود به صفحه اصلی، با اینکه استریم قابل مشاهده نیست، ولی صداش پخش میشه (یعنی فیلتری که درست کردم درست نیست)
خودتون پیشنهاد میدید چجوری فیلتر نویسی رو یاد بگیرم؟ آیا خودتون آموزش میدید یا آموزشی در سطح اینترنت سراغ دارید که مناسب من باشه؟

**Reply by MasterKia** [reply]
ازین منابع کمک بگیرید:

https://github.com/uBlockOrigin/uBlock-issues/wiki/

https://adblockplus.org/filter-cheatsheet

https://help.adblockplus.org/hc/en-us/articles/360062733293-How-to-write-filters

https://adguard.com/kb/general/ad-filtering/create-own-filters/

اگه سوالی داشتید بپرسید.

**Comment by waifucon** [comment]
اگر نت قطع شد و گیت‌هاب وایت‌لیست خورده بود اینجا رو یادتون نره 🙂

**Comment by LiterallyKG3** [comment]
rip
